gapis/atom/list.go

```diff
@@ -1,59 +0,0 @@
-// Copyright (C) 2017 Google Inc.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package atom
-
-import (
-	"context"
-
-	"github.com/google/gapid/gapis/api"
-)
-
-// List is a list of atoms.
-type List struct {
```

Comment:
Why do you want it? It made use awkward, and hid the fact we were making whole-capture copies of the entire command stream everywhere.